src/frontend/helpers/addToCart.js

```diff
@@ -16,9 +16,23 @@ const addToCart = async (product, setCart, token) => {
 
 		if (response.status === 201) {
 			setCart(response.data.cart);
+			toast.success("Product Added To Cart", {
+				position: "top-right",
+				autoClose: 1000,
+			});
 		}
 	} catch (error) {
-		console.error(error);
+		if (error.response.status === 500) {
+			toast.error("Please Login", {
+				position: "top-right",
+				autoClose: 1000,
+			});
+		} else {
+			toast.error("Failed to add item to Cart", {
```

Comment:
Instead of specifying the style on each toast, you can define the global style in toastContainer.

Reply:
Added this